covid19_sfbayarea/data/alameda/cases_by_age.py

```diff
@@ -0,0 +1,26 @@
+from typing import Any, Dict, List
+from .power_bi_querier import PowerBiQuerier
+
+class CasesByAge(PowerBiQuerier):
+    def __init__(self) -> None:
+        self.source = 'v'
+        self.name = 'V_Combined_data'
+        self.property = 'AgeGroup'
+        super().__init__()
+
+    def _parse_data(self, response_json: Dict[str, List]) -> List[Dict[str, int]]:
+        results = super()._parse_data(response_json)
+        return [ { 'group': group, 'raw_count': count } for group, count in results if group != 'Unknown Age' ]
```

Comment:
I’ve lost track and can’t check since you rebased; were you dropping the “Unknown Age” group before? Should we be doing that/is there a value there we should show?

Reply:
Yes - I had been dropping this before. There is a value there, but I don't know that it fits in with our frontend dashboard. Should we preserve it?

Reply:
My tendency is generally towards keeping it in unless we know it’s problematic to have (e.g. the issue with having per-ethnicity counts and totals together in the first review). BUT the right answer here is probably to ask the front-end folks when you’re not sure.

That said, it doesn’t seem like the decision should block getting this in. Just don’t let that be a reason to drop and not address it after. :)

Reply:
Assigning myself this as an issue so I can sync with the frontend team at hack night: https://github.com/sfbrigade/data-covid19-sfbayarea/issues/147